.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,7 +23,7 @@ Please delete options that are not relevant.
 - [ ] My code adheres to the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
 - [ ] I ran `make pylint` and code is rated 10/10
 - [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] I ran `make test` and all tests pass
+- [ ] I ran `make test testnb` and all tests pass
```

Comment:
Should we do something like make test-all or even have make test run all tests and then use another make command to test only .py files?

Reply:
As per chat, added `test-nb` and `test-py` for notebook and py file tests, and changed `test` to run them all.